README-automountmap.md

```diff
@@ -54,6 +54,19 @@ Example playbook to ensure presence of an automount map:
       desc: "this is a map for servers in the DMZ"
 ```
 
+Example playbook to ensure an indirect automount map is present:
```

Comment:
Please explain what an indirect map is and what it is used for.

Reply:
Fixed. I also added a new example playbook for indirect maps.